packages/catalog-generator/src/test/java/io/kaoto/camelcatalog/generator/CamelYamlDslSchemaProcessorTest.java

```diff
@@ -60,7 +62,41 @@ public void testExtractSingleOneOfFromAnyOf() throws Exception {
 
         assertTrue(errorHandlerSchema.has("oneOf"));
         assertTrue(errorHandlerSchema.get("oneOf").isArray());
-        assertEquals(7, errorHandlerSchema.get("oneOf").size());
+        assertEquals(6, errorHandlerSchema.get("oneOf").size());
+    }
+
+    @Test
+    public void testRemoveEmptyProperties() throws Exception {
```

Comment:
in JUnit 5, it is recommended to not use the the public modifier and keep default package
https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods